src/python_bindings/test_bindings.py

```diff
@@ -1,14 +1,212 @@
-from inspect import getmembers, isclass
-
-import desbordante
-
-for algorithm_name, type_ in getmembers(desbordante, isclass):
-    if not (issubclass(type_, desbordante.FdAlgorithm)
-            and type_ is not desbordante.FdAlgorithm):
-        continue
-    algorithm = type_()
-    for option_name in algorithm.get_possible_options():
-        print(option_name, algorithm.get_option_type(option_name))
-    algorithm.load_data('WDC_satellites.csv', ',', False)
-    algorithm.execute()
-    print(algorithm_name, algorithm.get_fds())
+from itertools import combinations
+
+import desbordante as desb
+import unittest
+
+
+class OptionContainer:
+    def __init__(self, path, load_options, execute_options) -> None:
+        self.load_options = load_options
+        self.execute_options = execute_options
+        self.path = path
+
+
+only_null_equal_null_option = OptionContainer(
+    "WDC_satellites.csv", {"is_null_equal_null": False}, {}
+)
+
+
+def get_common_option_container(execute_options):
+    return OptionContainer(
+        "WDC_satellites.csv", {"is_null_equal_null": False}, execute_options
+    )
+
+
+def get_apriori_load_container(load_options):
+    return OptionContainer("TestWide.csv", load_options, {})
+
+
+def check_metric_verifier_failure(dataset, **kwargs) -> bool:
```

Comment:
```suggestion
def check_metric_verifier_failure(dataset, options) -> bool:
```
It doesn't look like you need kwargs here